src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/HttpVersionUtils.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace Microsoft.PowerShell.Commands
+{
+    internal static class HttpVersionUtils
+    {
+        public static readonly IReadOnlyCollection<string> AllowedVersions;
+
+        static HttpVersionUtils()
+        {
+            var fields = typeof(HttpVersion).GetFields(System.Reflection.BindingFlags.Static | System.Reflection.BindingFlags.Public);
```

Comment:
Looks great but it is seem too expensive to use a reflection for so simple scenario. I'd prefer a static string array with well-know values.

Reply:
There is very small impact. Static ctor called only once (when tab pressed first time). On the other hand we've got dynamic completion list. If it still doesn't fit i'll fix it.

Reply:
I have no strict objections and leave this to other reviewers.